plugins/org.jkiss.dbeaver.ext.dameng/src/org/jkiss/dbeaver/ext/dameng/model/DamengPrivUser.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
```

Comment:
It is already 2024 :)

Reply:
2024, a new Year,the copyright has been updated